### DISCUSSION TITLE
Sobre impressão de documentos

### DISCUSSION THREAD

**Discussion by D1Alexandre** [TadaSoftware/PyNFe#234] [Q&A]
Olá acho que essa duvida vai ser minha utima ha ha ha ha, gostaria de saber se tem algum exemplo de impressão

**Comment by juniortada** [comment]
https://github.com/TadaSoftware/PyNFe/discussions/264
https://github.com/TadaSoftware/PyNFe/discussions/260

**Comment by leogregianin** [comment]
Veja: https://github.com/Engenere/BrazilFiscalReport
